src/client/pages/OfferNew/Introduction/Sidebar/DetailsModal/utils.test.ts

```diff
@@ -0,0 +1,249 @@
+    firstName: 'Karl Ove',
+    lastName: 'Knausgård',
+    birthDate: '1968-12-06',
+    norwegianTravel: {
+      coInsured: 1,
+      isYouth: false,
+    },
+  }
+
+  it('returns the correct quote details properties for Norwegian travel quote form', () => {
+    const norwegianTravelDetailsFormData = getQuoteDetailsFormData({
+      form: norwegianTravelForm,
+      offerData: mockedNoTravelOfferData,
+      quoteDetails: mockedNoTravelQuoteDetails!,
+    })
+    const hasCoInsured = 'coInsured' in norwegianTravelDetailsFormData!
+    const hasIsYouth = 'isYouth' in norwegianTravelDetailsFormData!
+
+    expect(norwegianTravelDetailsFormData).toBeTruthy()
+    expect(Object.keys(norwegianTravelDetailsFormData!).length).toBe(2)
+    expect(hasCoInsured).toBe(true)
+    expect(hasIsYouth).toBe(true)
+    expect(
+      (norwegianTravelDetailsFormData as EditNorwegianTravelInput)?.coInsured,
+    ).toBe(1)
+  })
+
+  it('returns the correct quote details properties for Norwegian combo quote form', () => {
+    const norwegianComboDetailsFormData = getQuoteDetailsFormData({
+      form: norwegianComboForm,
+      offerData: mockedNoComboOfferData,
+      quoteDetails: mockedNoHomeQuoteDetails!,
+    })
+    const hasStreet = 'street' in norwegianComboDetailsFormData!
+    const hasCoInsured = 'coInsured' in norwegianComboDetailsFormData!
+
+    expect(norwegianComboDetailsFormData).toBeTruthy()
+    expect(Object.keys(norwegianComboDetailsFormData!).length).toBe(6)
+    expect(hasStreet).toBe(true)
+    expect(hasCoInsured).toBe(true)
+    expect(
+      (norwegianComboDetailsFormData as EditNorwegianHomeContentsInput)
+        ?.livingSpace,
+    ).toBe(111)
+  })
+
+  it('returns the correct updated quote details properties for Swedish apartment quote form', () => {
+    const swedishApartmentDetailsFormData = getQuoteDetailsFormData({
+      form: swedishApartmentForm,
+      offerData: mockedSeApartmentOfferData,
+      quoteDetails: mockedSeApartmentQuote.quoteDetails,
+    })
+    const hasStreet = 'street' in swedishApartmentDetailsFormData!
+    const hasHouseholdSize = 'householdSize' in swedishApartmentDetailsFormData!
+
+    expect(swedishApartmentDetailsFormData).toBeTruthy()
+    expect(Object.keys(swedishApartmentDetailsFormData!).length).toBe(5)
+    expect(hasStreet).toBe(true)
+    expect(hasHouseholdSize).toBe(true)
+    expect(
+      (swedishApartmentDetailsFormData as EditSwedishApartmentInput)?.type,
+    ).toBe(ApartmentType.Brf)
+  })
+
+  it('returns the correct updated quote details properties for Danish home & accident bundle quote form', () => {
+    const danishHomeAccidentDetailsFormData = getQuoteDetailsFormData({
+      form: danishHomeAccidentForm,
+      offerData: mockedDkHomeAccidentOfferData,
+      quoteDetails: mockedDkHomeQuote!.quoteDetails,
+    })
+    const hasStreet = 'street' in danishHomeAccidentDetailsFormData!
+    const hasCoInsured = 'coInsured' in danishHomeAccidentDetailsFormData!
```

Comment:
Is there a particular reason to check for street and coInsured or just that we want to check some properties?

Reply:
No, just wanted to check for some properties and thought it might not be necessary to check for all of them, since we also check that the number of properties is correct? But maybe we check for all of them explicitly?

Reply:
I think it sufficient, we can always add more later. Just thought if I missed that it was something special with those two properties :)